log/sudoers_ts.ksy

```diff
@@ -0,0 +1,114 @@
+meta:
+  id: sudoers_ts
+  title: Sudoers Time Stamp file
+  license: CC0-1.0
+  endian: le
+doc: |
+  This spec can be used to parse sudo time stamp files located in directories
+  such as /run/sudo/ts/$USER or /var/lib/sudo/ts/$USER.
+doc-ref: https://www.sudo.ws/man/1.8.27/sudoers_timestamp.man.html
+seq:
+  - id: records
+    type: record
+    repeat: eos
+types:
+  record:
+    seq:
+      - id: version
+        doc: version number of the timestamp_entry struct
+        type: u2
+      - id: size
```

Comment:
Minor nitpick: [style guide](http://doc.kaitai.io/ksy_style_guide.html) recommends using `len_payload` for size of certain attribute (`payload`) in bytes.

Reply:
It is the length of the record, so `len_record` would be better. I will change it. Thanks for your review!